packages/devops/scripts/deployment/startup.sh

```diff
@@ -11,9 +11,19 @@ DIR=$(dirname "$0")
 export STAGE=$(${DIR}/../utility/getEC2TagValue.sh Stage)
 echo "Starting up instance for ${STAGE}"
 
-# Fetch and deploy the latest for each package based on the stage, including injecting environment
-# variables from parameter store into the .env file
-${HOME_DIRECTORY}/packages/devops/scripts/deployment/deployLatestRepositories.sh
+# Set the branch based on STAGE
+if [[ $STAGE == "production" ]]; then
+    BRANCH="master"
+else
+    BRANCH="$STAGE"
+fi
+
```

Comment:
I believe this is not required in this script

```suggestion
```

Reply:
Oh oops - actually it is required here, but not required in `checkoutLatest.sh`. Will update - good catch!